foca/models/config.py

```diff
@@ -435,7 +436,7 @@ class SpecConfig(FOCABaseConfig):
             in Connexion's `connexion.apps.flask_app` module.
 
     Attributes:
-        path: Path to an OpenAPI 2.x or 3.x specification in YAML format.
+        path: Path(s) to OpenAPI 2.x or 3.x specification in YAML format.
```

Comment:
Please keep the description in `Args` and `Attributes` section the same if a class/instance attribute with the same name exists.